config/jobs/etcd/etcd-operator-presubmits.yaml

```diff
@@ -31,11 +31,14 @@ presubmits:
             memory: "4Gi"
   - name: pull-etcd-operator-test-e2e
     optional: true # remove once the job is stable
-    cluster: eks-prow-build-cluster
+    cluster: k8s-infra-prow-build
```

Comment:
Is the theory that only the k8s-infra cluster has kind/dind support?

Reply:
Based on [config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml](https://github.com/kubernetes/test-infra/blob/6091241e19fd302f57213db43aae417a76654655/config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml#L3-L48), it seems like. The k8s-infra cluster is also on GCP, so my concern from cross-provider pulls for the GCR image would not be an issue anymore.

Any objection to not using that cluster? We're also using it for our ARM jobs already.